MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceViewQtGUI.cpp

```diff
@@ -177,7 +177,7 @@ void TomographyIfaceViewQtGUI::initLayout() {
   m_ui.tabMain->addTab(tabSetupW, QString("Setup"));
 
   // this is a Qt widget, let Qt manage the pointer
-  m_tabROIW = new ImageROIViewQtWidget(m_ui.tabMain);
+  m_tabROIW = new TomographyROIViewQtWidget(m_ui.tabMain);
```

Comment:
Could we rename this from `m_tabROIW` to `m_tabROIWidget` as initially I thought it was `m_tabROIWindow`?